web/components/home/Home.js

```diff
@@ -133,8 +190,28 @@ class Home extends React.Component {
           </div>
         </div>
 
+        <div className={`${ROW_STYLE}`}>
+          <h2 className="fill-width margin-bottom-xxl">Hacker stories</h2>
+          <div className="flex jc-between row-stories scale-row-tablet">
+            {
+              ['spencer', 'olivia', 'avinash'].map(person => (
+                <div className="scale-width-tablet fill-height" key={`${person}`}>
+                  <div className="relative overflow-hidden">
+                    {getImage(`${person}-bg`, userToBg[person], 'fill-width pad-right-s')}
+                    {getImage(`${person}`, userToImg[person], 'fill-width overlay')}
+                    <div className="overlay scale-hide-full-screen scale-hover-disable-full-screen hover-show fill-width fill-height">
```

Comment:
Should the hover effects be working right now?

Reply:
Yes, this PR is finalized.. is it not working for you? Also hover only works for 1900px or higher viewport width

Reply:
Ah I see it now, looks good! Is there a reason why it's enabled only for >1900px viewport width?

Reply:
Cant fit the text inside the box